src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/RethrowToPreserveStackDetails.cs

```diff
@@ -18,7 +18,7 @@ public abstract class RethrowToPreserveStackDetailsAnalyzer : DiagnosticAnalyzer
                                                                          s_localizableTitle,
                                                                          s_localizableMessage,
                                                                          DiagnosticCategory.Usage,
-                                                                         RuleLevel.IdeSuggestion,
+                                                                         RuleLevel.BuildWarning,
```

Comment:
Before we insert this change into the SDK, I recommend we dogfood this in couple of our repos once we have a new 
`Microsoft.CodeAnalysis.NetAnalyzers` package with this change:

1. Bump up the version dogfooded by roslyn repo and validate the new violations: https://github.com/dotnet/roslyn/blob/bbfee62c26d0172fa5e92c49b542ae649bfc365c/eng/Versions.props#L31
2. Bump up the version dogfooded by runtime repo and validate the new violations: https://github.com/dotnet/runtime/blob/88ab22d20acb21300c60969fe1c49f4d287bb780/eng/Analyzers.props#L9

Reply:
So runtime already has this on as a warning in their codebase. Running this against roslyn found no cases where it triggers. I will create PRs to update those repos but I do not need to make any code changes.